### DISCUSSION TITLE
Rpisurv perfect to display RTMP streams from Frigate on a TV

### DISCUSSION THREAD

**Discussion by tubalainen** [blakeblackshear/frigate#959] [Show and tell]
Take a look at: https://github.com/SvenVD/rpisurv/releases

Version 3.x beta now supports RTMP streams as well! Been running it for 2 years now and it works great.
The frame in the down right corner is the "best" image taken from the API in Frigate.
![image](https://user-images.githubusercontent.com/26795429/113266315-6ed3dd80-92d5-11eb-8838-cbe0232af2fb.png)

**Comment by tubalainen** [comment]
I have just posted a FR to implement MQTT to Rpisurv - so that it could automatically display the latest frigate event on the TV.
Unfortunately my coding skills are not on par to make a PR to Rpisurv. I am hoping for support from someone here that feels that this would be a cool option to implement! :)

https://www.tapatalk.com/groups/rpisurv/mqtt-and-or-cec-support-t115.html

**Comment by Minglarn** [comment]
Hi Sven! 
Trying to accoplish RTMP streams from Frigate to a dual screen RPI4 running RPISRURV.
However, no matter what I try it wont show the RTMP stream from frigate. 
Did you add some extra params to each RTMP stream...
Can you please show a snippet from 1 camera running the RTMP stream from frigate?

//Regards Minglarn...

**Reply by tubalainen** [reply]
> Hi Sven!
> Trying to accoplish RTMP streams from Frigate to a dual screen RPI4 running RPISRURV.
> However, no matter what I try it wont show the RTMP stream from frigate.
> Did you add some extra params to each RTMP stream...
> Can you please show a snippet from 1 camera running the RTMP stream from frigate?
> 
> //Regards Minglarn...

Tjenare - I am not SvenVD - I am Tubalainen.

I havnt tested RTMP my self, sorry. Just read that version 3 beta did "support RTMP", check with the guys at the rpisurv forum or post a bug on their git page.

:)

**Reply by Minglarn** [reply]
Tjäna! :)
Yes of course. 
Was going to write to Sven but saw your post. :) My brain must have been somewhere else at that moment.

**Comment by martinj9320** [comment]
@tubalainen I know it's been a while, but did you ever find a way to display the latest Frigate events on the TV? This is exactly what I'm currently trying to accomplish.

**Reply by tubalainen** [reply]
> @tubalainen I know it's been a while, but did you ever find a way to display the latest Frigate events on the TV? This is exactly what I'm currently trying to accomplish.

Hej! Yepyep. Sure did :)
Check out the link above, this one: https://www.tapatalk.com/groups/rpisurv/mqtt-and-or-cec-support-t115.html

The complete solution is there. I am sending an curl API call to that python hack "API" that pushed "virutal keys" on rpisurv to change different streams based on events coming from Frigate.

I am using the sub streams from the cameras (via go2rtc) for rpisurv.

Please tell me if I can be of any further assisstance. :)

**Reply by martinj9320** [reply]
Thank you! I missed that link. I'll look into it as soon as possible.
